# Definition of the service message structure

# Request
sensor_msgs/Image image

# Response
---
bool humans_detected
